### PR TITLE
[INLONG-3514][TubeMQ] Cluster query adds name and IP attributes

### PR DESCRIPTION
**Pull request by bluewang**
### Title Name: [INLONG-XYZ][component] Title of the pull request

where *XYZ* should be replaced by the actual issue number.

Fixes https://github.com/apache/incubator-inlong/issues/3514

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/service/MasterServiceImpl.java [hunk 1]
```diff
@@ -156,6 +156,21 @@ public MasterEntry getMasterNode(Long clusterId) {
         return masters;
     }
 
+    @Override
+    public List<MasterEntry> getMasterNodes(String masterIp) {
+        if (masterIp == null) {
+            return null;
+        }
+        List<MasterEntry> masters = masterRepository
+                .findMasterEntryByIpEquals(
```
Comment (healchow) [inline inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/service/MasterServiceImpl.java:165]: Why separate so many lines?

File: inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/service/interfaces/MasterService.java [hunk 2]
```diff
@@ -75,6 +75,13 @@
      */
     List<MasterEntry> getMasterNodes(Long clusterId);
 
+    /**
+     * get master in master ip
```
Comment (EMsnap) [inline inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/service/interfaces/MasterService.java:79]: get master by master ip

File: inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/cluster/ClusterController.java [hunk 3]
```diff
@@ -156,9 +166,9 @@ private TubeMQResult queryAllClusterVo() {
         List<ClusterEntry> allClusters = clusterService.getAllClusters();
         List<ClusterVo> clusterVos = Lists.newArrayList();
         for (ClusterEntry cluster : allClusters) {
-            MasterEntry masterNode = masterService.getMasterNode(cluster.getClusterId());
+            List<MasterEntry> masterNodes = masterService.getMasterNodes(cluster.getClusterId());
             ClusterVo allCount = getAllCount(Integer.valueOf((int) cluster.getClusterId()));
```
Comment (EMsnap) [inline inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/cluster/ClusterController.java:170]: getAllCount -> getCountInCluster

File: inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/service/MasterServiceImpl.java [hunk 4]
```diff
@@ -174,8 +184,10 @@ public TubeMQResult checkMasterNodeStatus(String masterIp, Integer masterWebPort
 
     @Override
     public String getQueryCountUrl(Integer clusterId, String method) {
-        MasterEntry masterEntry =
-                masterRepository.findMasterEntryByClusterIdEquals(clusterId);
+        List<MasterEntry> masterEntries =
+                masterRepository.findMasterEntriesByClusterIdEquals(clusterId);
+        int index = (int)(Math.random() * masterEntries.size());
+        MasterEntry masterEntry = masterEntries.get(index);
```
Comment (EMsnap) [inline inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/service/MasterServiceImpl.java:190]: just use getMasterNode in masterServiceImpl to get active master

**Comment by EMsnap** [comment]
PLAL

**Review by EMsnap — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**